rt/transports/http-hc/src/test/java/org/apache/cxf/transport/http/asyncclient/ActivatorTest.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.cxf.transport.http.asyncclient;
+
+import java.util.Collections;
+import java.util.Hashtable;
+import java.util.Map;
+
+import org.apache.cxf.Bus;
+import org.osgi.framework.BundleContext;
+
+import org.easymock.EasyMock;
+import org.junit.Test;
+
+public class ActivatorTest {
+
+    @Test
+    public void testConduitConfigurerUpdates() throws NoSuchMethodException {
```

Comment:
Thanks a lot for the fix, @jmylly, I think we could craft much simpler test case:

```
    @Test
    public void testConduitConfigurerUpdates() throws Exception {
        final Bus b = BusFactory.getThreadDefaultBus();
        final BundleContext context = EasyMock.mock(BundleContext.class);

        final ConduitConfigurer configurer = new ConduitConfigurer(context) {
            @Override
            public Object[] getServices() {
                return new Object[] {
                    b 
                };
            }
        };
        
        configurer.updated(new Hashtable<>());
    }
```

What do you think?

Reply:
Thanks, much cleaner than PartialMock. However, this test doesn't much verify the functionality. Pushed another version taking best of both.